presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java

```diff
@@ -97,8 +95,11 @@ public void run()
         @Option(name = "--environment", title = "environment", description = "the name of the environment to start", required = true)
         public String environment;
 
+        @Option(name = "--logs-dir", title = "logs dir", description = "location of the exported logs directory")
+        public String logsDirBase = "logs/";
```

Comment:
If i eg invoke test run from intellij directly, where will the logs go to?
the logs/ folder does not exist, right?

maybe it should be null by default, or tempdir?

(same for env up)

Reply:
to logs/ dir which is emptied on every run.

Reply:
At the very least it should be gitignored, but IMO (currently) not exporting logs by default is more reasonable.

Reply:
Yeah - I think it makes sense that default for `--logs-dir` should be null. And we handle that in a way that we do not bind any host directories for logs to containers.